content/desktop/install/windows-install.md

```diff
@@ -174,8 +174,8 @@ The `install` command accepts the following flags:
   - It must be used together with the `--allowed-org=<org name>` flag. 
   - For example:
 
-    ```text
-    --allowed-org=<org name> --admin-settings="{'configurationFileVersion': 2, 'enhancedContainerIsolation': {'value': true, 'locked': false}}"
+    ```console
```

Comment:
This isn't valid for console since it's just a flag. In this case we should either use text or the whole command